pants-plugins/sample_conf/rules.py

```diff
@@ -64,7 +64,7 @@ async def generate_sample_conf_via_fmt(
     pex = await Get(VenvPex, PexFromTargetsRequest, subsystem.pex_request())
 
     result = await Get(
-        FallibleProcessResult,
+        ProcessResult,
```

Comment:
This change makes prevents the `conf/st2.conf.sample` from being replaced if `tools/config_gen.py` exits with an error. It also makes pants report the error by printing stdout from the process, which includes the traceback.